pysages/collective_variables/coordinates.py

```diff
@@ -8,10 +8,40 @@
 
 
 def barycenter(positions):
+    """
+    Returns the geometric center, or centroid, of a group of points in space.
+
+    Parameters
+    ----------
+    positions : ndarray
+        Array containing the positions of the points for which to compute the barycenter.
+
+    Returns
+    -------
+    barycenter : ndarray
```

Comment:
In all of these `np` is actually `jax.numpy` so the return type is `DeviceArray` (or `jaxlib.xla_extension.DeviceArray`, but that is too verbose)

Reply:
Strictly speaking, for our collective variables all the inputs and outputs are actually `DeviceArray`s.